.travis.yml

```diff
@@ -111,12 +111,12 @@ script:
   - $COMPILER --version
   - ./autogen.sh
   - ./configure --enable-werror CC=$COMPILER
-  - make
-  - make -j9 check
-  - make -j9 distcheck
+  - make V=1
```

Comment:
instead of repeating `V=1` everywhere, would `export MAKEFLAGS=V=1` achieve the same goal?

- https://www.gnu.org/software/autoconf/manual/autoconf-2.61/html_node/The-Make-Macro-MAKEFLAGS.html